src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -33,7 +33,7 @@
 #ifdef _CubLog
 #define ARBORX_CubLog_save _CubLog
 #endif
-#define _CubLog
+#define _CubLog(format, ...)
```

Comment:
Do we even need to specify `format, `?

Reply:
No, but that is exactly how it looks like in the definition in `Thrust` so I thought keeping it would make sense.

Reply:
Fine by me.  Just curious if it works w/o.

Reply:
Yes, I tested it.